### PR TITLE
o1-feat/add older papers

### PR DESCRIPTION
**Pull request by OmaymaMahjoub**
## What?
Added some older MAS/RL papers.

### CODE REVIEW

File: Research Papers/Shallow learning/README.md [hunk 1]
```diff
@@ -261,12 +269,18 @@ This paper investigates a relatively new direction in Multiagent Reinforcement L
...
 
+<details> <summary> <a href="http://www.maths.lse.ac.uk/personal/stengel/TEXTE/geb1996a.pdf"> Efficient Computation of Behavior Strategies </a>by Bernhard von Stengel, 1994. <a href="link">  </a> </summary> We propose the sequence form as a new strategic description for an extensive game with perfect recall. It is similar to the normal form but has linear instead of exponential complexity and allows a direct representation and efficient computation of behavior strategies. Pure strategies and their mixed strategy probabilities are replaced by sequences of consecutive choices and their realization probabilities. A zero-sum game is solved by a corresponding linear program that has linear size in the size of the game tree. General two-person games are studied in the paper by Koller et al., 1996 (Games Econ. Behav. 14, 247–259). Journal of Economic Literature Classification Number: C72. <br> - </details>
```
Comment (arnupretorius) [inline Research Papers/Shallow learning/README.md:282]: Perhaps this should be under "extensive form games"?

File: Research Papers/Shallow learning/README.md [hunk 2]
```diff
@@ -297,6 +311,8 @@ This paper investigates a relatively new direction in Multiagent Reinforcement L
...
 
+<details> <summary> <a href="http://www.maths.lse.ac.uk/personal/stengel/TEXTE/nashsurvey.pdf"> Computing equilibria for two-person games </a>by R. J. Aumann and S. Hart, Elsevier, 2002. <a href="link">  </a> </summary> Appeared as Chapter 45, Handbook of Game Theory with Economic Applications, Vol. 3 (2002), eds. R. J. Aumann and S. Hart, Elsevier, Amsterdam, pages 1723–1759. This paper is a survey and exposition of linear methods for finding Nash equilibria. Above all, these apply to games with two players. In an equilibrium of a two-person game, the mixed strategy probabilities of one player equalize the expected payoffs for the pure strategies used by the other player. This defines an optimization problem with linear constraints. We do not consider nonlinear methods like simplicial subdivision for approximating fixed points, or systems of inequalities for higher-degree polynomials as they arise for noncooperative games with more than two players. These are surveyed in McKelvey and McLennan (1996). <br> - </details>
```
Comment (arnupretorius) [inline Research Papers/Shallow learning/README.md:314]: This seems more suited under the topic of "convergent learning"? Although there is no learning strategy employed, it is concerned with converging to some equilibrium. Maybe we can make a new category for pure game theoretic strategies?

**Review by arnupretorius — COMMENTED** [review]
Thanks @OmaymaMahjoub 🙂  

Just see my few comments.

File: Research Papers/Shallow learning/README.md (hunk 2 above)
Comment (OmaymaMahjoub) [reply Research Papers/Shallow learning/README.md:314]: How about we keep it under "convergent learning" and add #to-do and back to it if there are more than one paper related to pure game theoretic strategies?

File: Research Papers/Shallow learning/README.md (hunk 1 above)
Comment (OmaymaMahjoub) [reply Research Papers/Shallow learning/README.md:282]: Thanks will change it now :raised_hands:

**Pull request merged**